# Barometric Pressure reading from LPS22HB
# The temperature detection of the air pressure sensor is only used for compensation. 
# For accurate temperature detection, please observe the value of the SHTC3 temperature and humidity sensor.
float32 pressure
float32 temperature